Description: remove ATSPI roles that require a version of at-spi2-core newer
 than the package in xenial (2.18.3).
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/content/browser/accessibility/accessibility_tree_formatter_utils_auralinux.cc
+++ b/content/browser/accessibility/accessibility_tree_formatter_utils_auralinux.cc
@@ -105,9 +105,6 @@ CONTENT_EXPORT const char* ATSPIRoleToSt
       QUOTE(ATSPI_ROLE_COMMENT),
       QUOTE(ATSPI_ROLE_DATE_EDITOR),
       QUOTE(ATSPI_ROLE_DEFINITION),
-      QUOTE(ATSPI_ROLE_DESCRIPTION_LIST),
-      QUOTE(ATSPI_ROLE_DESCRIPTION_TERM),
-      QUOTE(ATSPI_ROLE_DESCRIPTION_VALUE),
       QUOTE(ATSPI_ROLE_DESKTOP_FRAME),
       QUOTE(ATSPI_ROLE_DESKTOP_ICON),
       QUOTE(ATSPI_ROLE_DIAL),
@@ -129,7 +126,6 @@ CONTENT_EXPORT const char* ATSPIRoleToSt
       QUOTE(ATSPI_ROLE_FOCUS_TRAVERSABLE),
       QUOTE(ATSPI_ROLE_FONT_CHOOSER),
       QUOTE(ATSPI_ROLE_FOOTER),
-      QUOTE(ATSPI_ROLE_FOOTNOTE),
       QUOTE(ATSPI_ROLE_FORM),
       QUOTE(ATSPI_ROLE_FRAME),
       QUOTE(ATSPI_ROLE_GLASS_PANE),
